libfs/remote_status.go

```diff
@@ -109,7 +107,7 @@ func (r *RemoteStatus) ExtraFileName() string {
 	r.Lock()
 	defer r.Unlock()
 
-	if r.extraFileName == "" || time.Since(r.failingSince) > failureDisplayThreshold {
+	if r.extraFileName == "" || time.Since(r.failingSince) < failureDisplayThreshold {
```

Comment:
We return "" until it has failed for awhile? As long as this is the intent, but it seems counterintuitive, like what if there is no error?